core/src/main/scala/sttp/tapir/server/interceptor/CustomInterceptors.scala

```diff
@@ -32,44 +32,44 @@ import sttp.tapir.{headers, statusCode}
   * @param decodeFailureHandler
   *   The decode failure handler, from which an interceptor will be created. Determines whether to respond when an input fails to decode.
   */
-case class CustomInterceptors[F[_], L, O](
-    createLogInterceptor: ServerLog[L] => Interceptor[F],
-    createOptions: CustomInterceptors[F, L, O] => O,
+case class CustomInterceptors[F[_], O](
+    createLogInterceptor: ServerLog[F] => Interceptor[F],
```

Comment:
maybe we could remove this parameter, and if a `ServerLog` is given, simply create the logging interceptor? If somebody wants to customise logging, they can provide a custom interceptor anyway